content/en/docs/tasks/job/automated-tasks-with-cron-jobs.md

```diff
@@ -170,13 +170,13 @@ After the deadline, the cron job does not start the job.
 Jobs that do not meet their deadline in this way count as failed jobs.
 If this field is not specified, the jobs have no deadline.
 
-The CronJob controller counts how many missed schedules happen for a cron job. If there are more than 100 missed schedules, the cron job is no longer scheduled. When `.spec.startingDeadlineSeconds` is not set, the CronJob controller counts missed schedules from `status.lastScheduleTime` until now. 
+The CronJob controller counts how many missed schedules happen for a cron job. If there are more than 100 missed schedules, the cron job is no longer scheduled. When `.spec.startingDeadlineSeconds` is not set, the CronJob controller counts missed schedules from `status.lastScheduleTime` until now.
```

Comment:
I think this is a change in the behavior for in the new controller. The default controller now raises a warning of how many schedules were missed but still schedules this. I wonder if we should change update this text accordingly?

Reply:
Good point, I just dropped it, it doesn't make sense to inform about warnings, and the older controller is deprecated. I don't want to also confuse users when I'll be talking about two controllers, either.